doc/api/resources/orders.rst

```diff
@@ -206,6 +206,17 @@ checkins                              list of objects            List of **succe
 ├ device                              integer                    Internal ID of the device. Can be ``null``. **Deprecated**, since this ID is not otherwise used in the API and is therefore not very useful.
 ├ device_id                           integer                    Attribute ``device_id`` of the device. Can be ``null``.
 └ auto_checked_in                     boolean                    Indicates if this check-in been performed automatically by the system
+print_logs                            list of objects            List of print jobs recorded e.g. by the pretix apps
+├ id                                  integer                    Internal ID of the print job
+├ successful                          boolean                    Whether the print job successfully resulted in a print.
+                                                                 This is not expected to be 100 % reliable information (since
+                                                                 printer feedback is never perfect) and there is no guarantee
+                                                                 that unsuccessful jobs will be logged.
```

Comment:
```suggestion
│                                                                This is not expected to be 100 % reliable information (since
│                                                                printer feedback is never perfect) and there is no guarantee
│                                                                that unsuccessful jobs will be logged.
```